### PR TITLE
Benefits section responsive, fixed eslint errors

### PR DESCRIPTION
**Pull request by george-usynin-deriv**

### CODE REVIEW

File: src/pages/index.tsx [hunk 1]
```diff
@@ -5,10 +5,9 @@ import { BenefitSection } from "../components/BenefitsSection/BenefitSection";
 const Home = () => {
     return (
         <div>
-            <Comprehensive id="derivApi"/>
-            Good luck Trolls!
+            <Comprehensive id="derivApi" />
             <BenefitSection />
-            <Comprehensive id="comprehensive"/>
```
Comment (Larkgronick) [inline src/pages/index.tsx]: Can id be equal?

File: src/components/BenefitsSection/BenefitSection.tsx [hunk 2]
```diff
@@ -12,6 +13,30 @@ export const BenefitSection = () => {
...
+                    <div className={`${styles.personalisation}`}>
+                        <h3>Personalisation</h3>
+                        <p>
+                            Personalise your trading applications to match your needs. Create charts and views just the
+                            way you like them. Develop your trading applications using any common programming language
+                            and extend your trading opportunity.
+                        </p>
+                    </div>
+                    <div className={`${styles["personalisation-image"]}`}>
+                        <Image alt="personalisation icon" src="/personalisation.png" width="486" height="260" />
```
Comment (Larkgronick) [inline src/components/BenefitsSection/BenefitSection.tsx:26]: We can use variables as image paths.

Comment (george-usynin-deriv) [reply src/components/BenefitsSection/BenefitSection.tsx:26]: we use static path with the appropriate name of the image in the Public folder

**Review by Larkgronick — APPROVED** [review]

File: src/pages/index.tsx (hunk 1 above)
Comment (george-usynin-deriv) [reply src/pages/index.tsx]: maryia.frantsava will refactor this code

**Pull request merged**